src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -53,44 +32,16 @@ const ResearchProfile = () => {
     ],
   });
 
-  const fullName = person?.names ? getFullCristinName(person.names) : '';
-  const orcidUri = getOrcidUri(person?.identifiers);
-  const activeAffiliations = person?.affiliations ? filterActiveAffiliations(person.affiliations) : [];
-
   return (
-    <StyledPageContent>
-      <PageHeader>{fullName}</PageHeader>
-      {isLoadingPerson || isLoadingRegistrations ? (
-        <PageSpinner aria-label={t('my_page.research_profile')} />
-      ) : (
-        person && (
+    <>
+      <Box sx={{ display: 'grid', gridTemplateColumns: '3fr 1fr', gap: '1rem' }}>
+        {isLoadingRegistrations ? (
+          <PageSpinner aria-label={t('my_page.research_profile')} />
+        ) : (
           <BackgroundDiv sx={isPublicPage ? undefined : { padding: 0 }}>
-            <Typography variant="h2">{t('common.employments')}</Typography>
-            {activeAffiliations.length > 0 && (
-              <Box sx={lineSx}>
-                <WorkIcon />
-                <Box sx={textContainerSx}>
-                  {activeAffiliations.map(({ organization }) => (
-                    <AffiliationHierarchy key={organization} unitUri={organization} commaSeparated />
-                  ))}
-                </Box>
-              </Box>
-            )}
-            {orcidUri && (
-              <Box sx={lineSx}>
-                <IconButton size="small" href={orcidUri} target="_blank">
-                  <img src={orcidIcon} height="20" alt="orcid" />
-                </IconButton>
-                <Box sx={textContainerSx}>
-                  <Typography component={MuiLink} href={orcidUri} target="_blank" rel="noopener noreferrer">
-                    {orcidUri}
-                  </Typography>
-                </Box>
-              </Box>
-            )}
             {registrations && (
-              <Box sx={{ mt: '2rem' }}>
-                <Typography variant="h2" gutterBottom>
+              <Box>
```

Comment:
En grunn til at vi trenger et DOM-element her?
```suggestion
              <>
```